types/user/resource.pp

```diff
@@ -18,7 +18,6 @@
     Optional[gid]                      => Accounts::User::Uid,
     Optional[group]                    => Accounts::User::Name,
     Optional[groups]                   => Array[Accounts::User::Name],
-    Optional[name]                     => Accounts::User::Name,
```

Comment:
is that actually a duplicate? I don't see it again

Reply:
During the time it took to get this PR approved, someone else made a commit that removed the duplicate entry.